### PR TITLE
(SIMP-MAINT) Java related cleanup

### PR DESCRIPTION
**Pull request by trevor-vaughan**
* Ensure that we don't drop a random file named 'echo' on the system
* Remove usage of 'simp_file_line' since we don't actually know what
  environment we are using at this point and the module path may not
  have this function loaded.

### CODE REVIEW

File: lib/simp/cli/commands/bootstrap.rb [hunk 1]
```diff
@@ -358,12 +365,27 @@ def configure_bootstrap_puppetserver
       FileUtils.chown(vardir_stat.uid, vardir_stat.gid, server_conf_tmp)
       FileUtils.chmod(vardir_stat.mode & 0777, server_conf_tmp)
 
-      # Java 8 dropped -XX:MaxPermSize
-      xx_args = (java_major_version && (java_major_version < 8)) ? ' -XX:MaxPermSize=256m' : ''
+      java_args = [
+        '-Xms2g',
+        '-Xmx2g',
```
Comment (judyj) [inline lib/simp/cli/commands/bootstrap.rb]: also, should this be repeated?

Comment (trevor-vaughan) [reply lib/simp/cli/commands/bootstrap.rb]: @judyj Those are two different values.

**Comment by trevor-vaughan** [comment]
@judyj I just rebased it but didn't have any issues with the test either time.

File: lib/simp/cli/commands/bootstrap.rb [hunk 2]
```diff
@@ -117,6 +119,15 @@ def run(args)
 
     fix_file_contexts
 
+    # After the first run the puppetserver will normally come up on a different port,
```
Comment (lnemsick-simp) [inline lib/simp/cli/commands/bootstrap.rb]: This comment only applies for FOSS version, not PE version

File: lib/simp/cli/commands/bootstrap.rb [hunk 3]
```diff
@@ -117,6 +119,15 @@ def run(args)
...
+    # reloading puppetserver to apply this change
+    #
+    # TODO: Validate that the pupmod-simp-pupmod tests are properly checking
```
Comment (lnemsick-simp) [inline lib/simp/cli/commands/bootstrap.rb]: Have you entered a ticket for this TODO

**Pull request merged**